src/components/tooltip/TrackPower.js

```diff
@@ -68,8 +71,9 @@ class TooltipTrackPowerImpl extends React.PureComponent<Props> {
   _computeCO2eFromPower(power: number): number {
     // total energy Wh to kWh
     const energy = power / 1000;
-    const { WORLD } = averageIntensity.data;
-    return energy * WORLD;
+    const intensity =
+      this.props.meta.gramsOfCO2ePerKWh || averageIntensity.data.WORLD;
```

Comment:
nit: use `??` instead of `||` so that we test for its absence rather than it being falsy. Indeed I guess it's theorically possible to have it set to 0 in case of 100% renewable.

Reply:
0 doesn't make sense, even for renewable. As an example, my renewable electricity provider estimates the carbon intensity of its electricity at 18.5g per kWh for production plus 5.15g per kWh for distribution. Using solar panels on my roof would have about the same footprint. If we wanted to support the 0 value, I would expect its meaning to be "hide everything in the UI that talks about CO2eq".